src/GraphQL/Buffers/BufferBase.php

```diff
@@ -138,9 +138,10 @@ protected function resolveBuffer(\SplObjectStorage $buffer) {
    *
    * @param array $buffer
    *   The buffer as an array.
-   *
+   * @codingStandardsIgnoreStart
```

Comment:
let's not ignore coding standards checks, rather keep rule excluded, so it can be fixed later

Reply:
this is still being here, let's remove this line and also `@codingStandardsIgnoreEnd` below